lol.janjic.Splitcat.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>lol.janjic.Splitcat</id>
+
+    <name>Splitcat</name>
+    <developer_name>Ivan Janjić</developer_name>
+    <summary>Simple application for splitting and merging files.</summary>
+
+    <metadata_license>MIT</metadata_license>
+    <project_license>MIT</project_license>
+
+    <recommends>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>touch</control>
+    </recommends>
+
+    <description>
+        <p>Simple file splitting and merging application with predefined presets for popular apps.</p>
+    </description>
+
+    <launchable type="desktop-id">lol.janjic.Splitcat.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/vogonwann/splitcat/refs/heads/main/screenshots/preset.png</image>
+            <caption>Presets screen</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/vogonwann/splitcat/refs/heads/main/screenshots/custom.png</image>
```

Comment:
the screenshots should refer to a specific commit not a branch.

Reply:
OK

Reply:
`https://github.com/vogonwann/splitcat/blob/9e7f57c242123f75c87eeff36db35dd4c5c259fe/screenshots/merge.png` 
is this OK?

Reply:
that's not the image. That's the github page.

Reply:
sorry... this?
`https://raw.githubusercontent.com/vogonwann/splitcat/9e7f57c242123f75c87eeff36db35dd4c5c259fe/screenshots/merge.png`

Reply:
Yes

Reply:
done